src/pages/myNotes/notes.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+
+import { Container, NotesList, StyledListComponent } from "./styled";
+
+import List from "@mui/material/List";
+import { ListItemText } from "@mui/material";
+
+import ComponentInfo from "./componentInfo";
+import NoteListItem from "./noteListItem";
+import { todoList } from "../../constans/noteList";
+
+const Notes = () => {
+  const [componentInfo, setComponentInfo] = useState(null);
+
+  const getItemInfo = (e) => {
+    const getCurrentItem = todoList.find((todo) => {
+      return todo.id == e.currentTarget.id;
+    });
+    setComponentInfo(getCurrentItem);
+    console.log(componentInfo);
+  };
+
+  return (
+    <NotesList>
+      <Container>
+        <List>
+          {todoList.map((todo) => (
+            <StyledListComponent
+              className={
+                componentInfo && componentInfo.id === todo.id ? "active" : null
```

Comment:
do you really need the class name here? Why?

Reply:
To change style of clicked element

Reply:
I see, it makes sense and I can accept it, but let's try to do the same through props. I mean try to pass "isActive" prop to the component and in ./styled add styles based on the prop

Reply:
Done